guides/v2.1/extension-dev-guide/cache/partial-caching/database-caching.md

```diff
@@ -87,24 +87,24 @@ To modify `di.xml`:
          	   </argument>
         	</arguments>
     	</type>
+      ```
 
     where `<your cache id>` is your unique cache identifier.
 
 5.	Save your changes to `di.xml` and exit the text editor.
 
-7.	Continue with <a href="#mage-cache-db-verify">Verify database caching is working</a>.
+7.	Continue with [Verify database caching is working].
 
-<h2 id="mage-cache-db-env">Database caching using a custom cache frontend</h2>
+## Database caching using a custom cache frontend {#mage-cache-db-env}
 This section discusses how to set up database caching with a custom {% glossarytooltip ca5ad9ac-9d39-45b5-80b1-e90d192f20d0 %}cache frontend{% endglossarytooltip %}.
 
-<div class="bs-callout bs-callout-info" id="info">
+{:.bs-callout .bs-callout-info #info}
 <span class="glyphicon-class">
```

Comment:
Remove HTML span element

Reply:
@leviv, I think I made this comment as you were making changes to the callout classes. Please remove span.